### PR TITLE
Require Java 11 to build

### LINKED ISSUE #27: Build requires JDK 9 or higher
And CI needs to be updated accordingly as well.

### PR DESCRIPTION
**Pull request by dmlloyd**
Fixes #27

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -98,6 +98,26 @@
...
+                <executions>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>11</version>
```
Comment (kenfinnigan) [inline pom.xml:113]: Any reason for the requirement?

It's still compiling against JDK 8 APIs right?

Comment (dmlloyd) [reply pom.xml:113]: It produces an MR JAR for at least one module, so you need to build with an appropriate JDK to support all the layers.  The base layer is Java 8 though.

**Comment by Ladicek** [comment]
The produced artifacts are still Java 8 compatible?

**Comment by Ladicek** [comment]
OK, I see the question was already answered in the comment. Thanks :-)

**Pull request merged**